src/extensions/common/hither/HitherJobStatusView.tsx

```diff
@@ -5,20 +5,20 @@ import React, { FunctionComponent } from 'react'
 import { HitherJob } from './HitherInterface'
 
 
-const HitherJobStatusView: FunctionComponent<{job?: HitherJob, width?: number, height?: number}> = ({job, width=200, height=200}) => {
+const HitherJobStatusView: FunctionComponent<{job?: HitherJob, message?: string, width?: number, height?: number}> = ({job, message='', width=200, height=200}) => {
```

Comment:
Allow passing a message to HitherJobStatusView (didn't end up using it after all, though)